2024-02-29 13:55:43,434 INFO    MainThread:4912 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-29 13:55:43,434 INFO    MainThread:4912 [wandb_setup.py:_flush():76] Configure stats pid to 4912
2024-02-29 13:55:43,434 INFO    MainThread:4912 [wandb_setup.py:_flush():76] Loading settings from C:\Users\flora\.config\wandb\settings
2024-02-29 13:55:43,434 INFO    MainThread:4912 [wandb_setup.py:_flush():76] Loading settings from C:\Users\flora\OneDrive\Documentos\MESTRADO_UFSC\rotinas\python\ml-class\my_data\wandb\settings
2024-02-29 13:55:43,435 INFO    MainThread:4912 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-29 13:55:43,435 INFO    MainThread:4912 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-29 13:55:43,435 INFO    MainThread:4912 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'my_data\\test_run_conv_mydata.py', 'program_abspath': 'C:\\Users\\flora\\OneDrive\\Documentos\\MESTRADO_UFSC\\rotinas\\python\\ml-class\\my_data\\test_run_conv_mydata.py', 'program': 'C:\\Users\\flora\\OneDrive\\Documentos\\MESTRADO_UFSC\\rotinas\\python\\ml-class\\my_data\\test_run_conv_mydata.py'}
2024-02-29 13:55:43,436 INFO    MainThread:4912 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\flora\OneDrive\Documentos\MESTRADO_UFSC\rotinas\python\ml-class\my_data\wandb\run-20240229_135543-f8xvvhza\logs\debug.log
2024-02-29 13:55:43,436 INFO    MainThread:4912 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\flora\OneDrive\Documentos\MESTRADO_UFSC\rotinas\python\ml-class\my_data\wandb\run-20240229_135543-f8xvvhza\logs\debug-internal.log
2024-02-29 13:55:43,436 INFO    MainThread:4912 [wandb_init.py:init():566] calling init triggers
2024-02-29 13:55:43,436 INFO    MainThread:4912 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-02-29 13:55:43,436 INFO    MainThread:4912 [wandb_init.py:init():611] wandb.init() called when a run is still active
2024-02-29 13:55:53,985 WARNING MsgRouterThr:4912 [router.py:message_loop():77] message_loop has been closed
